cmd/server/routes/employeesRouter.go

```diff
@@ -1,29 +1,25 @@
 package routes
```

Comment:
In the last class, Cyro told us that `snack_case` names are commonly used for files and folder. Do you think that would be interesting to change `employeesRouter` to `employees_router`?

Reply:
Done.